### PR TITLE
Adding a Bio for Alex

### PR DESCRIPTION
**Pull request by mckalexee**
This change adds a bio for Alex to the website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,8 @@
 <body>
   <h1>Cool Cats Coding!</h1>
   <p>OH BOY</p>
+  <h2>Alex McKenzie</h2>
```
Comment (mckalexee) [inline index.html:16]: I think h2 makes sense here, since h1 should only appear once per page.

File: index.html [hunk 2]
```diff
@@ -13,6 +13,8 @@
...
+  <p>More like a cool tiger.</p>
```
Comment (mckalexee) [inline index.html:17]: I was going back and forth between tiger and lion, but tiger made more sense.

**Review by mckalexee — COMMENTED** [review]
Self review complete. Your turn @jolyn1993 and @molliesmith

File: index.html (hunk 1 above)
Comment (jolyn1993) [reply index.html]: LGTM

**Review by molliesmith — APPROVED** [review]
LGTM

**Review by justinegab — APPROVED** [review]
LGTM

**Pull request merged**